assets/js/payment-method-extensions/payment-methods/cheque/index.js

```diff
@@ -12,7 +12,7 @@ import { decodeEntities } from '@wordpress/html-entities';
 import { PAYMENT_METHOD_NAME } from './constants';
 
 const settings = getSetting( 'cheque_data', {} );
-const defaultLabel = __( 'Check Payment', 'woo-gutenberg-products-block' );
+const defaultLabel = __( 'Check payment', 'woo-gutenberg-products-block' );
```

Comment:
I _think_ we should be using sentence case for all our labels, including payment methods. So I've fixed this one - happy to change to title case if needed.

Reply:
Absolutely! Both Gutenberg and Calypso are converging to sentence case, so we should too.